Gemfile.lock

```diff
@@ -74,10 +74,10 @@ GEM
       coderay (~> 1.1)
       method_source (~> 1.0)
     public_suffix (4.0.6)
-    puma (5.5.1)
+    puma (5.6.2)
       nio4r (~> 2.0)
     racc (1.5.2)
-    rack (2.2.3)
+    rack (1.6.13)
```

Comment:
@mheap this dependency update bumps `rack` down to the previous major version. Is this fine?